uniflow/op/transform/model_op.py

```diff
@@ -2,7 +2,7 @@
 import copy
 from typing import Sequence
 
-from uniflow.model.model import Model
+from uniflow.flow.transform.model import Model
```

Comment:
this looks like a bad idea to import from flow into op for model. I think model.py should be move under the transform folder.

Reply:
You mean model_op.py needs to be under the transform folder?

Reply:
fixed!